redux/action.js

```diff
@@ -8,6 +8,8 @@ import {
   DEL_FROM_SAVELATER,
 } from './actionTypes';
 
+const BASE_API_URL = process.env.NEXT_PUBLIC_BASE_API_URL;
```

Comment:
Nit: Can we have these values go into a constants file please? Easier to reuse them that way.